MovieDB/MovieDB/Scenes/Dashboard/DashboardCoordinator.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+protocol DashboardCoordinatorType {
+    func goToWatchList()
+    func goToMovieDetail(movieId: Int)
+    func goToSearch(query: String)
+}
+
+final class DashboardCoordinator: Coordinator, DashboardCoordinatorType {
+    var navigationController: UINavigationController?
+
+    init(navigationController: UINavigationController?) {
+        self.navigationController = navigationController
+    }
+
+    func start() {
+        var dashboardVC = DashboardViewController.instantiate()
+        let useCase = DashboardUseCase()
+        let viewModel = DashboardViewModel(coordinator: self,
+                                           useCase: useCase)
+        dashboardVC.bind(to: viewModel)
+        navigationController?.navigationBar.isHidden = true
+        navigationController?.pushViewController(dashboardVC, animated: true)
+    }
+
+    func finish() {
+    }
+    
+    func goToWatchList() {
+        //navigation to WatchList Screen
```

Comment:
If the func will be developed in future, should add TODO on comment 
// TODO: something note